web-apps/genesys-spark-examples/src/styles/component-viewer.scss

```diff
@@ -196,3 +205,12 @@ body {
   cursor: not-allowed;
   opacity: 0.9;
 }
+
+.editor-toggle {
+  display: flex;
+  justify-content: start;
+  padding: 2px 0;
+  margin-left: 16px;
+  font-size: 10px;
+  cursor: pointer;
```

Comment:
Removed some of these and defaulted the toggle to checked for visible because it kind of makes more sense. Monse from UX had a look at it and is ok with this version.